client/packages/common/src/ui/layout/tables/components/Cells/NumberInputCell/NonNegativeDecimalCell.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { CellProps } from '../../../columns';
+import { NonNegativeNumberInput } from '@common/components';
+import { RecordWithId } from '@common/types';
+import { useBufferState, useDebounceCallback } from '@common/hooks';
+
+// where NonNegative is n >=0
+export const NonNegativeDecimalCell = <T extends RecordWithId>({
```

Comment:
concerned about the duplication between this component and the `NonNegativeIntegerCell` but it is unnecessarily complicated to add in a prop for the value rounding. best to keep what you have I think